.github/ISSUE_TEMPLATE/github-sponsors-profile-review.yml

```diff
@@ -0,0 +1,16 @@
+name: GitHub Sponsors Profile Review
+description: Please review my GitHub Sponsors profile on a livestream
+title: "[REVIEW] <profile name here>"
+labels: [gh sponsor review, live stream review]

```

Comment:
Note: a new label should be created for this. :)

Reply:
<img width="113" alt="image" src="https://user-images.githubusercontent.com/63889819/185994043-87968759-0c80-488c-a978-fc1eca0ca12f.png">

Label is present. (:

Reply:
I remembered this time to create it first 👍  - thank you for checking